scripts/join_pane.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# where the secondary key bindings are bound
+SECONDARY_KEY_TABLE="join-pane"
+
+# more of a convenient sentinel for the full screen join
+BREAK_PANE_FLAG="-b"
+
+# passed to script on the 2-key binding invocation
+JOIN_PANE_FLAG="$1"
+
+source "$CURRENT_DIR/helpers.sh"
+
+join_pane() {
+  local join_pane_flag="$1"
+	tmux join-pane "$join_pane_flag" || \
+    tmux display-message "Mark a(nother) pane first"
```

Comment:
I see you're mixing tabs and spaces for indentation on the above 2 lines. Can we use tabs as they're used for all `tmux-plugins`?

Reply:
my editor auto converts things, but i did start playing around with the defaults there part way through

will put in conformant whitespace in the next iteration :+1: